openacademy/i18n/es.po

```diff
@@ -0,0 +1,291 @@
+#: model:ir.model.fields,field_description:openacademy.field_openacademy_course__session_ids
+#: model:ir.model.fields,field_description:openacademy.field_openacademy_wizard__session_ids
+#: model:ir.model.fields,field_description:openacademy.field_res_partner__session_ids
+#: model:ir.model.fields,field_description:openacademy.field_res_users__session_ids
+#: model:ir.ui.menu,name:openacademy.menu_session
+#: model:ir.ui.menu,name:openacademy.sub_menu_session
+#: model_terms:ir.ui.view,arch_db:openacademy.session_view_calendar
+msgid "Sessions"
+msgstr "Sesiones"
+
+#. module: openacademy
+#: model:ir.actions.act_window,name:openacademy.action_launch_wizard
+#: model:ir.ui.menu,name:openacademy.menu_wizard
+msgid "Sessions Wizard"
+msgstr "Asistente de Sesiones"
+
+#. module: openacademy
+#: model:ir.model.fields,field_description:openacademy.field_openacademy_session__start_date
+msgid "Start Date"
+msgstr "Fecha de Inicio"
+
+#. module: openacademy
+#: model_terms:ir.ui.view,arch_db:openacademy.session_report
+msgid "Start date"
+msgstr "Fecha de Inicio"
+
+#. module: openacademy
+#: model:ir.model.fields,field_description:openacademy.field_openacademy_session__taken_seats
+msgid "Taken Seats"
+msgstr "Asientos Ocupados"
+
+#. module: openacademy
+#: code:addons/openacademy/models/openacademy_session.py:0
+#, python-format
+msgid "The instructor cannot be attendee."
+msgstr "El instructor no puede ser participante."
+
+#. module: openacademy
+#: code:addons/openacademy/models/openacademy_course.py:0
+#: model:ir.model.constraint,message:openacademy.constraint_openacademy_course_unique_course_name
+#, python-format
+msgid "The name of the course must be unique."
+msgstr "El nombre del curso debe ser único."
+
+#. module: openacademy
+#: code:addons/openacademy/models/openacademy_session.py:0
+#, python-format
+msgid "The number of participants cannot be greater than the number of seats."
+msgstr "El número de participantes no puede ser superior al número de asientos."
+
+#. module: openacademy
+#: code:addons/openacademy/models/openacademy_session.py:0
+#, python-format
+msgid "The number of seats cannot be negative."
+msgstr "El número de asientos no puede ser negativo."
+
+#. module: openacademy
+#: code:addons/openacademy/models/openacademy_course.py:0
+#: model:ir.model.constraint,message:openacademy.constraint_openacademy_course_description_and_title_different
+#, python-format
+msgid "The title and description of the course must be different."
+msgstr "El título y la descripción del curso deben ser diferentes."
+
+#. module: openacademy
+#: model:ir.model,name:openacademy.model_openacademy_session
+msgid "The Session model manages the OpenAcademy sessions."
+msgstr "El modelo Sesión maneja las sesiones de OpenAcademy."
+
+#. module: openacademy
+#: model:ir.model,name:openacademy.model_openacademy_course
+msgid "The Course model manages the OpenAcademy courses."
```

Comment:
This is not sorted, "The course" should go before "The session". Please keep the same order as given by Odoo when exporting.

This will have a different order anyway, after you fix `_description`